deploy/helm/elastic-agent/examples/system-custom-auth-paths/agent-system-values.yaml

```diff
@@ -0,0 +1,9 @@
+system:
```

Comment:
We should have rendered examples for this, right? I thought the mage target would error in this case.

Reply:
hmm, I thought I added it but this never happened, but the CI should fail yes

Reply:
oh I suspect that this is because the file was never versioned and the `check-ci` apparently fails on changes of versioned files?!

Reply:
Shouldn't it fail anyway if it can't find the file? I didn't know the examples mage target was aware of versioning.

Anyway, this isn't a blocker for this PR, but we should make sure the check actually catches issues.

Reply:
the current CI checks flow is the following:

* render helm examples
* check for git diffs; the assumption here was that it failed also on unversioned files but evidently that's not the case

100% agreed this needs to be extended